lib/features/module_list_element_widget.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:udsp59/main.dart';
+
+class ModuleListElementWidget extends StatelessWidget {
```

Comment:
no need to suffix it with the `Widget` word. Everything is already a widget in Flutter